unittests/tools/test_dependency_track_parser.py

```diff
@@ -43,8 +43,10 @@ def test_dependency_track_parser_has_many_findings(self):
         testfile.close()
         self.assertEqual(4, len(findings))
 
-        self.assertIsNone(findings[0].unsaved_vulnerability_ids)
-        self.assertIsNone(findings[1].unsaved_vulnerability_ids)
+        self.assertEqual(1, len(findings[0].unsaved_vulnerability_ids))
+        self.assertEqual('533', findings[0].unsaved_vulnerability_ids[0])
```

Comment:
The vulnerability id is defined as "An id of a vulnerability in a security advisory associated with this finding. Can be a Common Vulnerabilities and Exposures (CVE) or from other sources." These examples are just numbers that don't fit to this definition.

Reply:
you mean you prefer to build "CWE-{x}" string?

Reply:
The vulnerability ids don't store CWEs but something like GHSA-57j2-w4cx-62h2 or CVE-2020-36518. They will contain the link to the source in the UI or could be googled.

Reply:
so we completly remove CWE from this field?